client/src/views/BlogView.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div :style='cssVars'>
+    <div class='title-section is-flex has-text-centered'>
+      <div class='h1-header-large'>
+        {{ title }}
+      </div>
+    </div>
+    <div class='section content' v-html='content'>
+    </div>
```

Comment:
nit: can you put this end tag on the previous line?

Reply:
Done!